### DISCUSSION TITLE
Jeti EXBus Telemetry

### DISCUSSION THREAD

**Discussion by jeti-flight** [iNavFlight/inav#6760] [Q&A]
Hello, I'm new to this repo and have read some older comments on Jeti Telemetry on INav in the internet.
Looking into the 2.7.0 code i can find a lot of things done for this telemetry but not (yet) activated. Is this somthing being worked on still? Can i offer help here? Of course would need some advice were to look and hoto test. I have Jeti equipment and Matek contollers (765/722) to test with.

**Comment by stronnag** [comment]
What is needed is:
* Split out the current monolithic Jeti implementation into the RX and telemetry as is done for the other RX protocols
* Update the inav telemetry to the capability of that in Betaflight

No one is working on this as none of developers has Jeti hardware, however for someone with (a) the hardware and (b) an understanding of the differences between inav and betaflight it should be fairly janitorial and straightforward.

**Reply by jeti-flight** [reply]
Ok, would todays monolithic implementation work to get familiar with the functions? If so i need a litte hint how to (re)enable and how it goes before splitting it out to the telemetry part.
